config/urls.py

```diff
@@ -48,7 +48,7 @@ def trigger_error(request):
 # Public URLs that are meant to be cached.
 public_urls = [
     path(
-        r"catalogue/id/<iaid:iaid>/",
+        r"catalogue/id/<iaid:metadata_id>/",
```

Comment:
The `iaid` here is the name of the registered converter. I initially thought to change it as `metadata_id` to match the API and URL param but reverted since the converter  serves the purpose of `iaid` 
The notion for `id` is updated in the ticket. Would that be relevant now?

Reply:
Going with discussions.